back/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+const port = 8000;
+const camping = require("./data").campingList
+
+const bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json());
+
+
+app.get('/', (request, response) => {
+  response.send('Welcome to Express');
+});
+
+app.get('/api/camping', (request, response) => {
+  response.send(camping);
+});
+
+app.post('/api/camping', (req, res) => {
```

Comment:
be consistent: always call your parameters req and res, or request and response, but don't mix :)